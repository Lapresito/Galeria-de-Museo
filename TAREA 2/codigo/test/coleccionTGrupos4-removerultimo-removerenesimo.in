crearColeccionGrupos
crearGrupoABB
crearVisitante 1 Zafiro Lopez 22
agregarAGrupoABB
agregarAColeccionGrupos
cantidadColeccionGrupos
removerUltimoColeccionGrupos
cantidadColeccionGrupos
crearGrupoABB
crearVisitante 2 Filemon Fernandez 32
agregarAGrupoABB
agregarAColeccionGrupos
crearGrupoABB
crearVisitante 3 Brunilda Sanchez 41
agregarAGrupoABB
agregarAColeccionGrupos
crearGrupoABB
crearVisitante 4 Filemon Martinez 44
agregarAGrupoABB
agregarAColeccionGrupos
removerNesimoColeccionGrupos 3
removerUltimoColeccionGrupos
cantidadColeccionGrupos
removerUltimoColeccionGrupos
crearGrupoABB
crearVisitante 10 Valquiria Perez 33
agregarAGrupoABB
agregarAColeccionGrupos
crearGrupoABB
crearVisitante 11 Ubaldo Lopez 53
agregarAGrupoABB
agregarAColeccionGrupos
crearGrupoABB
crearVisitante 12 Rufino Rodriguez 43
agregarAGrupoABB
agregarAColeccionGrupos
crearGrupoABB
crearVisitante 13 Valquiria Rodriguez 18
agregarAGrupoABB
agregarAColeccionGrupos
imprimirColeccionGrupos
removerNesimoColeccionGrupos 3
removerNesimoColeccionGrupos 1
removerNesimoColeccionGrupos 2
removerNesimoColeccionGrupos 1
liberarColeccionGrupos
Fin
