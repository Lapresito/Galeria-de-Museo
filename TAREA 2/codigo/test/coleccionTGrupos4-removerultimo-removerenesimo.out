Bienvenido al programa principal de ArteProg. Por favor ingrese su comando:
1> La coleccion de grupos ha sido creado de forma exitosa.
2> El grupoABB ha sido creado de forma exitosa.
3> Visitante creadox de forma exitosa.
4> Se ha agregado el visitante al grupo de forma exitosa.
5> Se ha agregado el grupo a la coleccion de forma exitosa.
6> La cantidad de grupos en la coleccion es 1
7> Ultimo grupo (removido):
Visitante Zafiro Lopez
Id: 1
Edad: 22
8> La cantidad de grupos en la coleccion es 0
9> El grupoABB ha sido creado de forma exitosa.
10> Visitante creadox de forma exitosa.
11> Se ha agregado el visitante al grupo de forma exitosa.
12> Se ha agregado el grupo a la coleccion de forma exitosa.
13> El grupoABB ha sido creado de forma exitosa.
14> Visitante creadox de forma exitosa.
15> Se ha agregado el visitante al grupo de forma exitosa.
16> Se ha agregado el grupo a la coleccion de forma exitosa.
17> El grupoABB ha sido creado de forma exitosa.
18> Visitante creadox de forma exitosa.
19> Se ha agregado el visitante al grupo de forma exitosa.
20> Se ha agregado el grupo a la coleccion de forma exitosa.
21> Grupo en la posicion 3 (removido):
Visitante Filemon Martinez
Id: 4
Edad: 44
22> Ultimo grupo (removido):
Visitante Brunilda Sanchez
Id: 3
Edad: 41
23> La cantidad de grupos en la coleccion es 1
24> Ultimo grupo (removido):
Visitante Filemon Fernandez
Id: 2
Edad: 32
25> El grupoABB ha sido creado de forma exitosa.
26> Visitante creadox de forma exitosa.
27> Se ha agregado el visitante al grupo de forma exitosa.
28> Se ha agregado el grupo a la coleccion de forma exitosa.
29> El grupoABB ha sido creado de forma exitosa.
30> Visitante creadox de forma exitosa.
31> Se ha agregado el visitante al grupo de forma exitosa.
32> Se ha agregado el grupo a la coleccion de forma exitosa.
33> El grupoABB ha sido creado de forma exitosa.
34> Visitante creadox de forma exitosa.
35> Se ha agregado el visitante al grupo de forma exitosa.
36> Se ha agregado el grupo a la coleccion de forma exitosa.
37> El grupoABB ha sido creado de forma exitosa.
38> Visitante creadox de forma exitosa.
39> Se ha agregado el visitante al grupo de forma exitosa.
40> Se ha agregado el grupo a la coleccion de forma exitosa.
41> Coleccion de grupos:
Grupo con edad promedio 18.00:
Visitante Valquiria Rodriguez
Id: 13
Edad: 18
Grupo con edad promedio 33.00:
Visitante Valquiria Perez
Id: 10
Edad: 33
Grupo con edad promedio 43.00:
Visitante Rufino Rodriguez
Id: 12
Edad: 43
Grupo con edad promedio 53.00:
Visitante Ubaldo Lopez
Id: 11
Edad: 53
42> Grupo en la posicion 3 (removido):
Visitante Rufino Rodriguez
Id: 12
Edad: 43
43> Grupo en la posicion 1 (removido):
Visitante Valquiria Rodriguez
Id: 13
Edad: 18
44> Grupo en la posicion 2 (removido):
Visitante Ubaldo Lopez
Id: 11
Edad: 53
45> Grupo en la posicion 1 (removido):
Visitante Valquiria Perez
Id: 10
Edad: 33
46> Coleccion de grupos liberada
47> Fin.
